interface/patient_file/report/custom_report.php

```diff
@@ -484,12 +484,17 @@ function next_prev(action){
   echo '<page_header style="text-align:right;" class="custom-tag"> ' . xlt("PATIENT") . ':' . text($titleres['lname']) . ', ' . text($titleres['fname']) . ' - ' . $titleres['DOB_TS'] . '</page_header>    ';
   echo '<page_footer style="text-align:right;" class="custom-tag">' . xlt('Generated on') . ' ' . oeFormatShortDate() . ' - ' . text($facility['name']) . ' ' . text($facility['phone']) . '</page_footer>';
 
-  // Use logo if it exists as 'practice_logo.png' in the site dir
-  // old code used the global custom dir which is no longer a valid
-   $practice_logo = "$OE_SITE_DIR/images/practice_logo.png";
-   if (file_exists($practice_logo)) {
-        echo "<img src='$practice_logo' align='left'><br />\n";
-     } 
+  //use logo if it exists in directory
+  if (file_exists("$OE_SITE_DIR/images/practice_logo.png"))  $practice_logo = "$OE_SITE_DIR/images/practice_logo.png";
+  else if (file_exists("$OE_SITE_DIR/images/practice_logo.gif"))  $practice_logo = "$OE_SITE_DIR/images/practice_logo.gif";
+  else if (file_exists("$OE_SITE_DIR/images/practice_logo.jpg"))  $practice_logo = "$OE_SITE_DIR/images/practice_logo.jpg";
+  else if (file_exists("$OE_SITE_DIR/images/practice_logo.jpeg")) $practice_logo =  "$OE_SITE_DIR/images/practice_logo.jpeg";
+  else $practice_logo = '';
+   $dir = explode('/', $practice_logo) ;
+   $practice_logo = "$dir[4]/$dir[5]/$dir[6]/$dir[7]/$dir[8]" ;
+   if ($practice_logo) {
+        echo "<img src='../../../../$practice_logo' align='left' style='width : 56px ; height : 90.6px'><br />\n";
```

Comment:
Could a variable be added to the globals like $OE_SITE_DIR to get this. The hard coding of the path information will cause issues down the road when things are changes and someone forgets to change this.

Reply:
Should be a global config to use an optional file specifically for this purpose...not sure if we even need a logo on a report like this at all.
  My final vote is that either we make the effort to put in a letterhead feature, or get rid of the image here.  It does nothing for the quality of the report.

Reply:
I also suggest that there is no specific need of image in the report.

Reply:
Abhinav,
Could you just do us all a favor and get rid of the image completely for now?  That is the simple and sensible solution.  It should have never been there in the first place.
Thanks,
-Art

Reply:
@aethelwulffe I suggest two options 
1)To completely remove the image.
2)To put a general medical logo like a plus sign or something like that.
Which one you favor?

Reply:
Totally remove the logo is my preference, but I am open to either option.

Reply:
Thank you for telling I will remove it.

Reply:
Just waiting on the new commit, and then I will test and merge.  I imagine this has been a very irritating issue for you.  Sorry such a simple thing got so complex.